src/Runtime/Eval.hs

```diff
@@ -54,7 +54,8 @@ bind (BVal (Sig n _) defs _) = (n, do
          Just (Vboard b) -> return $ Vboard (fill b sz defs values)
          _ -> error "TODO")
    where
-      newBoard sz = array ((1,1), sz) (zip [(x,y) | x <- [1..(fst sz)], y <- [1..(snd sz)]] (repeat (Vs "?"))) -- TODO: replace ?
```

Comment:
Know anything about this TODO?

Reply:
Both left by me, so mostly yeah! The first one at line 55 is something that will never occur. This is a board definition, so if the name ever referenced something that is not a board, then that could only have happened because the name occurs in a different definition and then the parser should have caught that. The most idiomatic way to handle this is to write to code so that the type system rules out this "impossible" (so I say...) case, but I am not clever enough yet ;).

The second one (57) is there because we never made a clear decision on how to denote un-initialized board cells. I made a temporary decision to use the symbol "?," but this was not meant to be permanent. However, instead of tucking it away as a TODO in some comment, I will make an issue out of it.